Library/Formula/capstone.rb

```diff
@@ -2,8 +2,8 @@
 
 class Capstone < Formula
   homepage "http://capstone-engine.org"
-  url "http://capstone-engine.org/download/3.0/capstone-3.0.tgz"
-  sha1 "26e591b8323ed3f6e92637d7ee953cb505687efa"
+  url "http://capstone-engine.org/download/3.0.1/capstone-3.0.1.tgz"
+  sha1 "7f206c853c6b8d05de22e919c2455ab661654093"
   revision 1
```

Comment:
Yup. No need to submit a new PR. You can make the change locally and then rebase and push; If you don't have the time or aren't that familiar with Git happy to talk you through it, or don't worry about it too much if you don't have the time :).